### PR TITLE
AL/MIN-354. Move tests from runtime.

### PR DESCRIPTION
**Pull request by lut1k**
**Description:**
 
Moved tests from runtime in pallets.

**Changes in storage or API:**

--

**Checklist**

- [x] Code is covered with tests
- [x] Code is covered with benchmarks
- [x] Fresh benchmark results included

### CODE REVIEW

File: pallets/traits/src/lib.rs [hunk 1]
```diff
@@ -183,7 +183,7 @@ pub trait MntManager<AccountId> {
 }
 
 /// An abstraction of risk-manager basic functionalities.
-pub trait RiskManagerAPI {
+pub trait RiskManagerManager {
```
Comment (kurotych) [inline pallets/traits/src/lib.rs:186]: `RiskManagerManager` ?

**Review by kurotych — COMMENTED** [review]
I would like to return ONE_HUNDRED and remove  extra zeroes from test.
This approach will save place

File: pallets/dex/src/tests.rs [hunk 2]
```diff
@@ -1,15 +1,90 @@
 //! Unit tests for dex module.
 
 #![cfg(test)]
-
 use crate::mock::*;
+use frame_support::assert_err;
+use orml_traits::MultiCurrency;
+use pallet_traits::DEXManager;
+
+fn liquidation_pool_balance(pool_id: CurrencyId) -> Balance {
```
[the remark this answers is not in the document]
Comment (lut1k) [reply pallets/dex/src/tests.rs:9]: Fixed

File: pallets/traits/src/lib.rs (hunk 1 above)
Comment (lut1k) [reply pallets/traits/src/lib.rs:186]: Fixed

**Comment by lut1k** [comment]
> I would like to return ONE_HUNDRED and remove extra zeroes from test.
> This approach will save place

I have reduced the length of the constant names. But I didn't change the numbers in the tests due to the fact that there are too many changes.

**Review by kurotych — DISMISSED** [review]

File: test-helper/src/lib.rs [hunk 3]
```diff
@@ -2,21 +2,63 @@
...
+
+	pub type AccountId = u64;
+
+	pub const ADMIN: AccountId = 0;
+	pub const ALICE: AccountId = 1;
+	pub const BOB: AccountId = 2;
+	pub const CHARLIE: AccountId = 3;
+
+	pub fn admin<Origin: OriginTrait<AccountId = AccountId>>() -> Origin {
+		Origin::signed(ADMIN)
```
Comment (kurotych) [inline test-helper/src/lib.rs:50]: let's name it admin_origin and so on.

Comment (lut1k) [reply test-helper/src/lib.rs:50]: Fixed

**Review by kurotych — APPROVED** [review]

**Review by Apollo50 — APPROVED** [review]
Looks good for me!

**Pull request merged**